src/routes/cart.router.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  addArticleToCart,
+  clearCart,
+  createCart,
+  getCart,
+  quantityArticleCart,
+  removeArticleFromCart,
+} from '@controlers/cart.controller';
+import { isAuth } from '@utils/isAuth';
+import { Router } from 'express';
+
+const cartRouter = Router();
+
+cartRouter.get('/cart', isAuth, getCart);
+cartRouter.post('/cart', isAuth, createCart);
+cartRouter.post('/add-cart', isAuth, addArticleToCart);
+cartRouter.post('/quantity-cart', isAuth, quantityArticleCart);
```

Comment:
why is there 2 routes that does almost same thing /add-cart and  /quantity-cart.
Make one route that can handle both cases because code inside is almost same